components/brave_wallet_ui/utils/solana-instruction-utils.ts

```diff
@@ -151,7 +151,13 @@ export const getSolInstructionAccountParamsObj = (
   let newAccount: string = ''
 
   accountParams.forEach(({ name }, i) => {
-    const value = accountMetas[i]?.pubkey
+    // Do not show account public key of address table lookup account because
```

Comment:
good call out here! Is there ever a time where it would be useful for a user to be able to look up the table lookup address even if it's not possible by default?

Reply:
Sounds good, I'm not seeing any blockers here and I'm +1 to the transaction simulation UI approach rather than manually parsing info, especially in Solana where the way they pass info between different accounts can be alot more complex.